DPG reference catalog for open standards

```diff
@@ -0,0 +1,54 @@
+###DPG Reference catalog for open standards
+##This reference catalog is an overview of standards that are recommended for the Digital Public Goods. The standards can be technical or semantic, and are sorted by relevant application. 
+
+#Datasets:
+JSON (JavaScript Object Notation)
+XML
+
+#Character encoding and languages:
+UTF-8, ISO/IEC 10646
+ISO 639-1/639-2/639-3
+
+#Web standards:
+HTML 
+CSS
+SSL
+HTTP
+XHTML (eXtensible HTML v1.0)
+FTP
+DNS
+SMTP
+
+#Accessibility:
+WCAG 2.0/2.1 (Web Content Accessibility Guidelines)
+
+#Link and reference
+Uniform Resource Identifier (URI)
+
+#Document formats:
+OOXML (Office Open XML) 
+ODF (Open Document Format) 
+PDF (PDF/A-1)
+EPUB 3.0 (For ebooks)
+
+#Multimedia:
+SVG (Scalable Vector Graphics) 
+PNG (Portable Network Graphics) 
+JPEG (Joint Photographic Experts Group) 
+Ogg MP3 (Moving Picture Experts Group: Audio Layer III) 
+FLAC (Free Lossless Audio Codac) 
+H.264 (H.264/MPEG-4 AVC) 
+AAC (Advanced Audio Coding) 
+MP4 (MPEG-4 Part 14)
+Codebase:
+Public Code
+FAIR Guiding Principles for scientific data management and stewardship
+
+#Design: 
+SDLC
+SSDLC
+
+#Information security: 
+OWASP/ ASVS,
```

Comment:
Any specific version of the ASVS that people should use? Is there a specific Level within the ASVS that people should be aiming for with respect to their project(s)? Should complementary guides such as the (mobile) MASVS or OWASP's IoT evaluation framework also be mentioned now that they are being split out from the main ASVS?

Reply:
FYI @nathanbaleeta